src/index.ts

```diff
@@ -159,9 +159,9 @@ class LnMessage {
     this.connectionStatus$.next('connecting')
     this._attemptReconnect = attemptReconnect
 
-    this.socket = new (
-      typeof window === 'undefined' ? (await import('ws')).default : window.WebSocket
-    )(this.wsUrl)
+    // https://stackoverflow.com/questions/17575790/environment-detection-node-js-or-browser#comment120214255_31456668
+    const isNode = typeof process !== 'undefined' && process?.versions?.node
+    this.socket = new (isNode ? (await import('ws')).default : WebSocket)(this.wsUrl)
```

Comment:
I think this should use `globalThis` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis

maybe:
```suggestion
    this.socket =  new (typeof globalThis.WebSocket === 'undefined' ? (await import('ws')).default : globalThis.WebSocket)(this.wsUrl)
```